### PR TITLE
glib 2.70.4

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by carlocab — APPROVED** [review]
Don't really know what the deal with `v8` on Linux is, but too many other formulae on both OSes rely on this one for `v8` to block this for too long.

Anyone want to take a look, @Homebrew/linux?

**Comment by iMichka** [comment]
Yes, can I have a day or two for this? Looks like a new error. I would like to get this fixed before we merge this PR.

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
Anyway the issue is unrelated to this PR, I'll continue separate investigations.

**Pull request closed without merging**
